### PR TITLE
fix(account.office): hide office offer for some subsidiaries

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

Disable Office attach option for some subsidiaries

MBP-147

Related : https://github.com/ovh-ux/ovh-module-exchange/pull/217

### CODE REVIEW

**Comment by euhmeuh** [comment]
Got it! You can use `FeatureAvailabilityService`

**Comment by cbourgois** [comment]
> Got it! You can use `FeatureAvailabilityService`

Please, don't : https://github.com/ovh-ux/ovh-manager-dedicated/pull/802

File: src/exchange/account/home/account-home.component.js [hunk 1]
```diff
@@ -82,10 +86,17 @@
...
 
     fetchCanUserSubscribeToOfficeAttach() {
-      return this.officeAttach
-        .retrievingIfUserAlreadyHasSubscribed(this.$routerParams.productId)
-        .then((userHasAlreadySubscribed) => {
-          this.userCanSubscribeToOfficeAttach = !userHasAlreadySubscribed;
+      return this.OvhApiMe.v6().get().$promise
+        .then(({ ovhSubsidiary }) => {
+          if (['CA'].includes(ovhSubsidiary)) {
+            return this.$q.when(true);
```
Comment (FredericEspiau) [inline src/exchange/account/home/account-home.component.js]:
```suggestion
            return this.$q.when(false);
```

User can't subscribe if they are from CA

**Comment by antleblanc** [comment]
@marie-j Pay attention that this fix should be also applied to `develop` branch.

**Pull request merged**